Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -51,4 +58,10 @@ extern u32 position_matrix_index[4];
 
 // VB_HAS_X. Bitmask telling what vertex components are present.
 extern u32 g_current_components;
+
+extern BitSet8 g_main_vat_dirty, g_preprocess_vat_dirty;
```

Comment:
Ideally place variables on their own line to clearly separate them from one another for reading.